crates/bin/c8y_configuration_poc/src/mqtt_ext.rs

```diff
@@ -0,0 +1,180 @@
+use async_trait::async_trait;
+use mqtt_channel::MqttError;
+use mqtt_channel::SinkExt;
+use mqtt_channel::StreamExt;
+use mqtt_channel::TopicFilter;
+use tedge_actors::mpsc::channel;
+use tedge_actors::mpsc::Receiver;
+use tedge_actors::mpsc::Sender;
+use tedge_actors::Actor;
+use tedge_actors::ActorBuilder;
+use tedge_actors::ChannelError;
+use tedge_actors::DynSender;
+use tedge_actors::LinkError;
+use tedge_actors::MessageBox;
+use tedge_actors::PeerLinker;
+use tedge_actors::RuntimeError;
+use tedge_actors::RuntimeHandle;
+
+pub type MqttMessage = mqtt_channel::Message;
+
+#[derive(Debug)]
+pub enum MqttActorMessage {
+    DirectMessage(mqtt_channel::Message),
+    PeerMessage(mqtt_channel::Message),
```

Comment:
Since the MqttMessage type is defined above, can't we use it as below?
```suggestion
    PeerMessage(MqttMessage),
```

Reply:
Yes, this would be clearer. => Will do.